manifests/components/prometheus.jsonnet

```diff
@@ -202,6 +202,8 @@ local get_cm_web_hook_url = function(port, path) (
             storage: "%dMi" % $.storage,
           },
         },
+        replicas: 1,
+        updateStrategy: { type: "RollingUpdate" },
```

Comment:
@anguslees this change gets the rollout status to be reported cleanly. Do you see any issues with this update?